client/blocks/reader-post-card/byline.jsx

```diff
@@ -93,25 +97,24 @@ class PostByline extends React.Component {
 					siteUrl={ streamUrl }
 					isCompact={ true } />
 				<div className="reader-post-card__byline-details">
-					<div className="reader-post-card__byline-author-site">
-						{ shouldDisplayAuthor &&
-						<ReaderAuthorLink
-							className="reader-post-card__link"
-							author={ post.author }
-							siteUrl={ streamUrl }
-							post={ post }>
-							{ post.author.name }
-						</ReaderAuthorLink>
-						}
-						{ shouldDisplayAuthor && showSiteName && ', ' }
-						{ showSiteName && <ReaderSiteStreamLink
-							className="reader-post-card__site reader-post-card__link"
-							feedId={ feedId }
-							siteId={ siteId }
-							post={ post }>
-							{ siteName }
-						</ReaderSiteStreamLink> }
-					</div>
+					{ shouldDisplayAuthor && <div className="reader-post-card__byline-author-site">
```

Comment:
we also use this component in the following and search streams, where we do want to keep the site info even if we suppress the author info. maybe update the check to `( shouldDisplayAuthor || showSiteName ) &&` ?